.github/workflows/CI.yml

```diff
@@ -161,6 +161,11 @@ jobs:
         run: |
           pytest -v --doctest-glob="docs/*.rst" docs/
 
+      - name: Run notebooks in docs
+        if: ${{ matrix.rdkit == true && matrix.openeye == true }}
+        run: |
+          pytest -v --nbval --sanitize-with docs/nbval-sanitize.cfg --ignore docs/_build/ docs/
```

Comment:
I picked up the `--nbval-lax` argument purely out of habit years ago, so I'm not sure which is right. Based on the current documentation, I'm pretty sure we could go with either, given that I don't know of any `#NBVAL_CHECK_OUTPUT` comments. I'd be nice to be standardized, though, and I think we should standardize on `--nbval`.